.gitignore

```diff
@@ -22,6 +22,6 @@ build/
 key.json
 
 test.conf
-kcbq-connector/src/integration-test/resources/test.properties
+test.properties
```

Comment:
Did this need to be changed here? Just in the interest of keeping these PRs limited to the GCS stuff, we probably don't want this change here, unless I'm missing something.